components/atom/tooltip/src/index.js

```diff
@@ -86,6 +86,12 @@ class AtomTooltip extends Component {
     target.oncontextmenu = this.handleContextMenu
     target.addEventListener('mouseover', this.disableTitle)
     target.addEventListener('mouseout', this.restoreTitle)
+
+    // We are emulating a client event to force display
+    if (target && this.props.alwaysShown) {
```

Comment:
I think this emulated behavior can change the default behavior of form elements focussing when  some vertical page loads. I ignore if there is some product setting by default some input ready to be completed when some page loads.